Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1.4
 # This needs to be bullseye-slim because the Ruby image is built on bullseye-slim
-ARG NODE_VERSION="16.18.1-bullseye-slim"
```

Comment:
We should keep specific version in the Dockerfile so we can explicitly trigger update (manual or auto update) when a new version is released.

Reply:
(Also it permits to know which version is used)

Reply:
Since CircleCI doesn't support the floating major, I went down to the minor level. The images are already built on every push to the `main` branch, so I don't think there needs to be the full version here just to trigger a release. That would happen during a normal release